perception/sandbox/polygrasp/src/polygrasp/segmentation_rpc.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import a0
+import numpy as np
+
+from polygrasp import serdes
+
+topic_key = "segmentation"
+
+log = logging.getLogger(__name__)
+
+
+class SegmentationClient:
+    def __init__(self):
+        self.client = a0.RpcClient(topic_key)
+
+    def segment_img(self, rgbd, min_mask_size=2500):
+        state = []
+
+        def onreply(pkt):
+            state.append(pkt.payload)
+
+        bits = serdes.rgbd_to_capnp(rgbd).to_bytes()
+        self.client.send(bits, onreply)
+
+        while not state:
+            pass
```

Comment:
Please avoid busy loops without some form of yield. This will effectively subtract a cpu core.

```py
def segment_img(self, rgbd, min_mask_size=2500):
    bits = serdes.rgbd_to_capnp(rgbd).to_bytes()
    state = self.client.send_blocking(bits)

    obj_masked_rgbds = []
    ...
```